libs/protocol-ui/src/stories/atoms/FormLabel.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import { Story } from '@storybook/react';
+import React from 'react';
+import {FormLabel, FormLabelProps} from '../../'
+
+export default {
+    title: 'Components/Atoms/FormLabel',
+    component: FormLabel,
+    argTypes: {
+        label:{
+            control: { type: 'text' },
+        },
+        //color: {control:'color'},
+    },
+};
+
+const Template: Story<FormLabelProps> = (args) => <FormLabel {...args} />;
+    
+const Default = Template.bind({})
+Default.args = {
+    label: 'DAO',
+    //color: 'gray.800'
```

Comment:
Can add this back in so it'll use the `gray.800` default value

Reply:
I see you uncommented that line 21. Thanks!